### ISSUE TITLE
Shift position of code sample filename and click-to-copy icon

### ISSUE THREAD

**Issue by tiwudickson** [kubernetes/website#48233]
Hey there I was reading the documentations and some how I find the text does not look properly aligned and wasn't looking good at all. I am attaching the screenshot for better understanding 

![image](https://github.com/user-attachments/assets/e3cf1198-45bd-4792-9a10-de0a888018e9)

as you can see in the shared screenshot the deployment.yaml is aligned in right but it should be aligned in left

**Comment by utkarsh-singh1** [comment]
Page -> https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#:~:text=Example:%20Deploying%20PHP%20Guestbook

/language en

**Comment by utkarsh-singh1** [comment]
@tiwudickson, I don't think this alignment is available only on this page, it is available across all of the example across all of the k8s docs.

**Comment by utkarsh-singh1** [comment]
Also, Changing the position of title from right to left alignment is a discussion among the kubernetes docs community, after it can be changed or not.

**Comment by sftim** [comment]
The filename being aligned top right doesn't seem like a bug to me. If it stands out: that could even be a good thing.

How is it a problem for readers, in your opinion, @tiwudickson?

/retitle Shift position of code sample filename and click-to-copy icon
/kind feature
/area web-development

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/48950
